crates/router/src/utils/verification.rs

```diff
@@ -0,0 +1,102 @@
+use actix_web::web;
+#[cfg(all(feature = "olap", feature = "kms"))]
+use api_models::verifications::{self, ApplepayMerchantResponse};
+use error_stack::{Report, ResultExt};
+#[cfg(feature = "kms")]
+use external_services::kms;
+
+use crate::{
+    core::errors::{self, api_error_response},
+    headers, logger,
+    routes::AppState,
+    services, types, utils,
+};
+
+const APPLEPAY_MERCHANT_VERIFICATION_URL: &str =
+    "https://apple-pay-gateway.apple.com/paymentservices/registerMerchant";
+const APPLEPAY_INTERNAL_MERCHANT_IDENTIFIER: &str = "merchant.com.noon.juspay";
```

Comment:
Can you check, is `APPLEPAY_INTERNAL_MERCHANT_IDENTIFIER` requires KMS encryption?

Reply:
That wouldn't require it as anyways it is an internal api and will be helpfull as it is for debugging purposes